src/disk.c

```diff
@@ -740,6 +778,12 @@ static int disk_read(void) {
   derive_t io_time = 0;
   derive_t weighted_time = 0;
   derive_t diff_io_time = 0;
+  derive_t discard_complete = 0;
+  derive_t discard_merged = 0;
+  derive_t discard_sectors = 0;
+  derive_t discard_time = 0;
+  derive_t flush_complete = 0;
+  derive_t flush_time = 0;
```

Comment:
You could add link to doc above the `fopen("/proc/diskstats", "r")` call:
`// https://www.kernel.org/doc/Documentation/ABI/testing/procfs-diskstats`